src/components/missionsItem.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import styles from './styles/missionpage.module.css';
+import {
+  ReserveMissions,
+  RemoveReserveMissions,
+} from './redux/mission/missionsSlice';
+
+function MissionItems({
+  missionId, mission, description, reserved,
+}) {
+  const dispatch = useDispatch();
+  const reservedStat = reserved;
+
+  const handleRemoveReserved = (e, missionId) => {
+    e.preventDefault();
+    dispatch(RemoveReserveMissions(missionId));
+  };
+
+  const handleReservedMissions = (e, missionId) => {
+    e.preventDefault();
+    dispatch(ReserveMissions(missionId));
+  };
+
+  return (
+    <tr className={styles.missionList}>
+      <td className={styles.mission_name}>{mission}</td>
+      <td>{description}</td>
+      <td className={styles.membership_stat}>
+        {reservedStat === true ? (
+          <button id={missionId} type="button" className={styles.active_member}>
+            Active member
+          </button>
+        ) : (
+          <button id={missionId} type="button" className={styles.not_member}>
+            Not a member
+          </button>
+        )}
+      </td>
+      <td className={styles.reservation_stat}>
+        {reservedStat === true ? (
+          <button
+            type="button"
+            onClick={(e) => handleRemoveReserved(e, missionId)}
+            className={styles.leave_mission}
+          >
+            Leave Mission
+          </button>
+        ) : (
+          <button
+            type="button"
+            onClick={(e) => handleReservedMissions(e, missionId)}
+            className={styles.join_mission}
+          >
+            Join Mission
+          </button>
+        )}
+      </td>
+    </tr>
+  );
+}
+
+MissionItems.propTypes = {
+  missionId: PropTypes.string.isRequired,
```

Comment:
- [ ] I recommend defining a default value for `missionId` to avoid issues and test errors 👇🏽 

```javascript
console.error
      Warning: Failed prop type: The prop `missionId` is marked as required in `MissionItems`, but its value is `undefined`.
```
```javascript
Snapshot Summary
 › 4 snapshots written from 4 test suites.

Test Suites: 1 failed, 4 passed, 5 total
Tests:       1 failed, 4 passed, 5 total
Snapshots:   4 written, 4 total
Time:        4.181 s
Ran all test suites.
```

Reply:
You did well fixing the changes recommended by the previous reviewer, however, kindly use a valid `prop type` which in this case is `number` so that the `console.log` problem is fixed 😄 

![image](https://github.com/geekelo/space-travelers-hub/assets/20537712/517900da-1011-44b9-97cc-669da2fb8773)